internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -621,17 +621,23 @@ func msgCID(msg *types.UnsignedMessage) cid.Cid {
 
 func makeEpostMessage(blockMiner address.Address) internalMessage {
 	return internalMessage{
-		miner:  blockMiner,
-		from:   vmaddr.SystemAddress,
-		to:     blockMiner,
-		value:  big.Zero(),
-		method: storagemining.ProcessVerifiedElectionPoStMethodID,
+		miner: blockMiner,
+		from:  vmaddr.SystemAddress,
+		to:    blockMiner,
+		value: big.Zero(),
+		// Dragons: figure out the new spec-actors place for this
+		// method: storagemining.ProcessVerifiedElectionPoStMethodID,
```

Comment:
This is no longer necessary. You can remove this method and the sending of the epost message.